src/timetracker/pages/404-page/404-page.jsx

```diff
@@ -0,0 +1,12 @@
+import BrokenRobot from '../../assets/broken.png'
+export default function PageNotFound() {
+    return (
+        <div className="min-h-screen body text-white">
+            <div className="flex flex-col items-center">
```

Comment:
I would also add text-center so it doesn't default to left aligned on smaller screens